postcode_lookup/templates/includes/postal_votes.html

```diff
@@ -0,0 +1,53 @@
+<{{ section.context.htag_primary }} id="{{toc_id}}">{% trans %}Voting by post{% endtrans %}</{{ section.context.htag_primary }}>
+{% if section.context.can_register %}
+    <{{ section.context.htag_secondary }}>{% trans %}Apply for a postal vote{% endtrans %}</{{ section.context.htag_secondary }}>
+    <p>
+      {% trans %}
+        If you know that you won't be able to get to the polling station on polling day, you can vote by post.
+      {% endtrans %}
+      {% trans date=section.timetable.postal_vote_application_deadline|date_filter %}
+        You have until {{ date }}.
+      {% endtrans %}
+    </p>
+    <p>
+      <a href="https://www.gov.uk/apply-postal-vote" class="o-button o-button--primary o-button--dark o-external-link">
+        {% trans %}Apply for a postal vote{% endtrans %}
+      </a>
+    </p>
+    <p>
+      {% trans %}
+        <a href="https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-post">Read more</a> about voting by post.
+      {% endtrans %}
+    </p>
```

Comment:
On the site, this looks like

![Screenshot at 2025-04-03 09-14-45](https://github.com/user-attachments/assets/950b7163-1806-4591-95aa-6c394e3f4b05)


With the button-link pointing to https://www.gov.uk/apply-postal-vote
and the link-link pointing to https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-post

Open to suggestions on other ways to lay this out though. We could move it up into the paragraph above the button?